.gitignore

```diff
@@ -38,3 +38,6 @@ pytestdebug.log
 *.idea
 *.iml
 *.secrets
+
+# generated markdown file with resource versions, will not be commited until we have mechanism to keep updated
+cfn-resources/resource-versions.md
```

Comment:
Can we run the scrip as part of an action that keeps the markdown update?

Reply:
A ticket was created to follow up on what you are mentioning https://jira.mongodb.org/browse/INTMDB-1217.